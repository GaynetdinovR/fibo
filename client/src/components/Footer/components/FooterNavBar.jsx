import styles from "../../../styles/components/Footer.module.sass";
import H6 from "../../../ui/H6.jsx";
import LinkInList from "../../../ui/LinkInList.jsx";

const FooterNavBar = () => {
	const links = ["YouTube", "Facebook", "Instagram", "ВКонтакте"];

	return (
		<nav className={styles.footer__nav}>
			<H6 className={styles.footer__nav_title}>Мы в соцсетях</H6>
			<ul className={styles.footer__nav_list}>
				{links.map((item, i) => (
					<LinkInList key={i} liClass={styles.footer__nav_item} aClass={styles.footer__nav_link}>
						{item}
					</LinkInList>
				))}
				<li className={styles.footer__nav_item}>
					<address className={styles.footer__nav_address}>
						<span> Москва ул. Проспект </span>
						<span> Вернадского 86В </span>
					</address>
				</li>
			</ul>
		</nav>
	);
};

export default FooterNavBar;