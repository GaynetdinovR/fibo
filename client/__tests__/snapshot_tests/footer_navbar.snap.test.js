import renderer from "react-test-renderer";
import { expect, it } from "@jest/globals";
import FooterNavBar from "../../src/components/Footer/components/FooterNavBar.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<FooterNavBar />).toJSON();
	expect(tree).toMatchSnapshot();
});
