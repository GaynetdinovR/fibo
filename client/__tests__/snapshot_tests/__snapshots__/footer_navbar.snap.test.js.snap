// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="footer__nav"
>
  <h6
    className="footer__nav_title h6"
  >
    Мы в соцсетях
  </h6>
  <ul
    className="footer__nav_list"
  >
    <li
      className="footer__nav_item link_in_list__item"
    >
      <a
        className="footer__nav_link link_in_list__link"
      >
        YouTube
      </a>
    </li>
    <li
      className="footer__nav_item link_in_list__item"
    >
      <a
        className="footer__nav_link link_in_list__link"
      >
        Facebook
      </a>
    </li>
    <li
      className="footer__nav_item link_in_list__item"
    >
      <a
        className="footer__nav_link link_in_list__link"
      >
        Instagram
      </a>
    </li>
    <li
      className="footer__nav_item link_in_list__item"
    >
      <a
        className="footer__nav_link link_in_list__link"
      >
        ВКонтакте
      </a>
    </li>
    <li
      className="footer__nav_item"
    >
      <address
        className="footer__nav_address"
      >
        <span>
           Москва ул. Проспект 
        </span>
        <span>
           Вернадского 86В 
        </span>
      </address>
    </li>
  </ul>
</nav>
`;
